### ISSUE TITLE
Use a code formatter to improve code consistency

### ISSUE THREAD

**Issue by quythanh** [leho-dev/mpc-extension#3]
The code in this project is currently formatted in 2-space indent. Meanwhile, most of editors i'm using keep the indentation as 4-space, and i'm also familiar with 4-space indent. I know, coding style is very personal, and I won't force my views on anybody, but this can make it more difficult to collaborate on the project.

I propose that we use a code formatter to improve the consistency of the code. For this project, I would recommend using Prettier, a popular and well-maintained code formatter that supports a variety of programming languages. It is also easy to use and configure.

In short, just add the `.prettierrc` file, or any other formatting tool.

**Comment by holedev** [comment]
Sorry, my bad. `.prettierrc` has been added following the request. 

Best regards,
leho-dev
